webapp/app/country/countryState.js

```diff
@@ -10,6 +10,9 @@ const keys = {
   config: 'config',
   status: 'status',
   countries: 'countries',
+  regions: 'regions',
+  countryRegions: 'countryRegions',
+  regionCountries: 'regionCountries',
```

Comment:
what's the difference between countryRegions and regionCountries?

Reply:
countryRegions means in which regions a country belongs, for ex. Finland can be part of many different regions.
regionCountries is the other way around - what countries belong to a region, ex. EU contains FIN but also other countries

Reply:
but why not to fetch regions together with the countries? so in the UI we have only countries with a property regionCodes (array) ? + the regions themselves .
would make this things easier ?

Reply:
why these 3 properties? can't you filter client side instead fetching all these?

Reply:
please answer this

Reply:
This was fixed, missed few keys.
Removing